elfpy/data/convert_data.py

```diff
@@ -267,43 +293,73 @@ def get_wallet_info(
                     tokenValue=num_base_token,
                 )
             )
-        # Query and add hyperdrive tokens to walletinfo
+
+        # Query and add LP tokens to wallet info
+        lp_token_prefix = int(hyperdrive_interface.AssetIdPrefix.LP)
```

Comment:
I think it is the same, but the safe way to do this is `hyperdrive_interface.AssetIdPrefix.LP.value`, per the [docs](https://docs.python.org/3/library/enum.html#enum.Enum.value). My understanding is that this is better bc it keeps the intended type of the enum.